velox/docs/functions/spark/regexp.rst

```diff
@@ -34,3 +34,44 @@ See https://github.com/google/re2/wiki/Syntax for more information.
     the entire string by anchoring the pattern using ``^`` and ``$``. ::
 
         SELECT rlike('1a 2b 14m', '\d+b'); -- true
+
+.. spark:function:: regex_replace(string, pattern, overwrite) -> varchar
+
+    Replaces all substrings in ``string`` that match the regular expression ``pattern`` with the string ``overwrite``. If no match is found, the original string is returned as is.
+    There is a limit to the number of unique regexes to be compiled per function call (kMaxCompiledRegexes defined in velox/functions/lib/Re2Functions.h).
+
+    Parameters:
+
+    - **string**: The string to be searched.
```

Comment:
Would you generate docs locally, verify the formatting and share the screenshot?

Reply:
<img width="939" alt="image" src="https://github.com/facebookincubator/velox/assets/14911410/572ae4f4-a155-4e8f-a5c3-3f2c51ecdf51">

Reply:
Added the screenshot of docs. Please let me know if this is as expected

Reply:
one minor comment here - please delete line56 
:noindex: 

as it results in unnecessary tabbing of the regex replace section

Reply:
@codyschierbeck Looks like there is an extra indentation for the description of the second function.

Reply:
Looks like the :noindex: line was causing that indent difference. Removed it and now it looks as follows.

![MicrosoftTeams-image](https://github.com/facebookincubator/velox/assets/14911410/d62b8dd2-a9f4-4203-9321-56c4b65943b9)

Reply:
I did some digging and found the original issue. The :noindex: line had one less space than the rest of the lines that proceeded it, pushing them back an entire indent. The problem was fixed by adding that additional space. Squashed this change and the incorrect change together with the general doc changes.

Reply:
@mbasmanova Below is a screenshot of the docs after adding the single space

<img width="949" alt="image" src="https://github.com/facebookincubator/velox/assets/14911410/473f81d0-5621-45a1-9495-b0898d1cda6f">

Reply:
Looks nice. Thank you.